src/crashReporter.js

```diff
@@ -7,28 +7,30 @@ const SentryNode = loadable.lib(() =>
   import(/* webpackChunkName: "s_entry-browser" */ '@sentry/browser'),
 );
 
-const crashReporter = (store) => (next) => (action) => {
-  try {
-    return next(action);
-  } catch (error) {
-    if (
-      __SENTRY__?.SENTRY_DSN ||
-      process?.env?.RAZZLE_SENTRY_DSN ||
-      window?.env?.RAZZLE_SENTRY_DSN
-    ) {
-      const loader = __CLIENT__ ? SentryBrowser : SentryNode;
-      loader.load().then((Sentry) => {
-        Sentry.withScope((scope) => {
-          scope.setExtras({
-            action,
-            state: store.getState(),
+const crashReporter = (middlewares) => [
```

Comment:
Please use the old style, as a pure Redux middleware. Don't reference the `middlewares` in this module. 

The reason is: we want to keep, as far as possible the "Volto domain" and "Redux domain" clean and separate, to make it easier to reason about them.